dev-lang/rust/files/1.74.1-cross-compile-libz.patch

```diff
@@ -0,0 +1,15 @@
+https://paste.sr.ht/~kchibisov/682321e0fd4a3ece4a4b7b71591896f5cd3cdb22
+
+diff --git a/compiler/rustc_llvm/build.rs b/compiler/rustc_llvm/build.rs
+index f606fa483ca..60440fa8eb5 100644
+--- a/compiler/rustc_llvm/build.rs
++++ b/compiler/rustc_llvm/build.rs
+@@ -240,6 +240,8 @@ fn main() {
+
+     if !is_crossed {
+         cmd.arg("--system-libs");
++    } else {
++        println!("cargo:rustc-link-lib=z");
+     }
```

Comment:
Have you tried always passing the `--system-libs` btw? Or is it just always broken, etc?

Reply:
Ah yes, I just tested this and simply removing the `if` conditional also succeeds.  I'll change the patch to that.